test/instances/eeth/EETHHyperdrive.t.sol

```diff
@@ -0,0 +1,620 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity 0.8.20;
+
+import { ILiquidityPool } from "contracts/src/interfaces/ILiquidityPool.sol";
+import { IEETH } from "contracts/src/interfaces/IEETH.sol";
+import { EETHHyperdriveCoreDeployer } from "contracts/src/deployers/eeth/EETHHyperdriveCoreDeployer.sol";
+import { EETHHyperdriveDeployerCoordinator } from "contracts/src/deployers/eeth/EETHHyperdriveDeployerCoordinator.sol";
+import { EETHTarget0Deployer } from "contracts/src/deployers/eeth/EETHTarget0Deployer.sol";
+import { EETHTarget1Deployer } from "contracts/src/deployers/eeth/EETHTarget1Deployer.sol";
+import { EETHTarget2Deployer } from "contracts/src/deployers/eeth/EETHTarget2Deployer.sol";
+import { EETHTarget3Deployer } from "contracts/src/deployers/eeth/EETHTarget3Deployer.sol";
+import { EETHTarget4Deployer } from "contracts/src/deployers/eeth/EETHTarget4Deployer.sol";
+import { HyperdriveFactory } from "contracts/src/factory/HyperdriveFactory.sol";
+import { EETHConversions } from "contracts/src/instances/eeth/EETHConversions.sol";
+import { IERC20 } from "contracts/src/interfaces/IERC20.sol";
+import { IHyperdrive } from "contracts/src/interfaces/IHyperdrive.sol";
+import { IEETHHyperdrive } from "contracts/src/interfaces/IEETHHyperdrive.sol";
+import { AssetId } from "contracts/src/libraries/AssetId.sol";
+import { ETH } from "contracts/src/libraries/Constants.sol";
+import { FixedPointMath, ONE } from "contracts/src/libraries/FixedPointMath.sol";
+import { HyperdriveMath } from "contracts/src/libraries/HyperdriveMath.sol";
+import { ERC20ForwarderFactory } from "contracts/src/token/ERC20ForwarderFactory.sol";
+import { ERC20Mintable } from "contracts/test/ERC20Mintable.sol";
+import { InstanceTest } from "test/utils/InstanceTest.sol";
+import { HyperdriveUtils } from "test/utils/HyperdriveUtils.sol";
+import { Lib } from "test/utils/Lib.sol";
+
+contract EETHHyperdriveTest is InstanceTest {
+    using FixedPointMath for uint256;
+    using HyperdriveUtils for IHyperdrive;
+    using Lib for *;
+
+    // The mainnet liquidity pool.
+    ILiquidityPool internal constant POOL =
+        ILiquidityPool(0x308861A430be4cce5502d0A12724771Fc6DaF216);
+
+    // The mainnet EETH token.
+    IEETH internal constant EETH =
+        IEETH(0x35fA164735182de50811E8e2E824cFb9B6118ac2);
+
+    // The mainnet address that has the ability to call the rebase function.
+    address internal constant MEMBERSHIP_MANAGER =
+        0x3d320286E014C3e1ce99Af6d6B00f0C1D63E3000;
+
+    // Whale accounts.
+    address internal EETH_TOKEN_WHALE =
+        address(0xCd5fE23C85820F7B72D0926FC9b05b43E359b7ee);
```

Comment:
Do we need to cast this? It looks like we don't on line 43. Either way, I say we just keep it consistent. I'm mostly surprised that you didn't need to cast.